src/daipecore/widgets/Widget.py

```diff
@@ -0,0 +1,38 @@
+from abc import ABC, abstractmethod
+import re
+
+
+class Widget(ABC):
+    @abstractmethod
```

Comment:
Není název týhle třídy trošku zavádějící? Jako Widget si představuju konkrétně ten jeden widget např. text, takže tady bych očekával, že od abstraktní třídy Widget bude dědit -> TextWidget, SelectWidget, atd.
Nemělo by to tedy spíš být Widgets.add_text()?

Reply:
Good point, souhlasim s tebou - melo by to byt spise Widgets, opravim a force pushnu. Diky za feedback!